api/krusty/extendedpatch_test.go

```diff
@@ -1213,3 +1066,44 @@ spec:
     app: busybox
 `)
 }
+
+func TestTargetMissingPatchJson6902Error(t *testing.T) {
+	th := kusttest_test.MakeHarness(t)
+	makeCommonFileForExtendedPatchTest(th)
+	th.WriteK("base", `
+resources:
+- servicemonitor.yaml
+patchesJson6902:
+- target:
+    group: monitoring.coreos.com
+    kind: ServiceMonitor
+    name: starboard-exporter
+    namespace: starboard
+    version: v2
+  path: patch.0.yaml
+`)
+	th.WriteF("base/servicemonitor.yaml", `
```

Comment:
Can we use the existing files in `makeCommonFileForExtendedPatchTest` for this test?